src/Lean/Server/Watchdog.lean

```diff
@@ -205,7 +233,7 @@ section ServerM
     let task ← IO.asTask (loop $ ←read) Task.Priority.dedicated
     task.map $ fun
       | Except.ok ev   => ev
-      | Except.error e => WorkerEvent.crashed e
+      | Except.error e => WorkerEvent.ioError e
```

Comment:
@mhuisi since we already `catch` worker crashes, I thought this should be a separate kind of `ioError` event. It may appear when `errorPendingRequests` fails to communicate with the client.